music21/abcFormat/__init__.py

```diff
@@ -1639,16 +1656,16 @@ def parse(self, forceKeySignature=None, forceDefaultQuarterLength=None):
 
 # ------------------------------------------------------------------------------
 class ABCHandler:
-
     # divide elements of a character stream into objects and handle
     # store in a list, and pass global information to compontns
-    def __init__(self):
-        # tokens are ABC objects in a linear stream
+    def __init__(self, abcVersion=None):
+        # tokens are ABC objects import n a linear stream
+        self.abcVersion = abcVersion # At this point, only "2.1" and None are valid settings.
+        self.abcDirectives = dict()
```

Comment:
please use literal `{}` instead of `dict()`

Reply:
Can abcVersion be a tuple like other version information (2, 1) -- or converted to such.  We should be doing > and < with versions.

Reply:
Done.